components/avatar/avatar.css

```diff
@@ -0,0 +1,32 @@
+/* Avatar */
+.avatar {
+  height: 50px;
```

Comment:
Try to use rem units

Reply:
Thank you for suggestion, I will change it to rem.